app/src/main/java/com/hadiyarajesh/notex/ui/reminders/RemindersScreen.kt

```diff
@@ -1,33 +1,206 @@
 package com.hadiyarajesh.notex.ui.reminders
 
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.padding
-import androidx.compose.material3.ExperimentalMaterial3Api
-import androidx.compose.material3.Scaffold
-import androidx.compose.material3.Text
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
+import androidx.paging.LoadState
+import androidx.paging.compose.LazyPagingItems
+import androidx.paging.compose.collectAsLazyPagingItems
+import androidx.paging.compose.items
 import com.hadiyarajesh.notex.R
+import com.hadiyarajesh.notex.database.entity.Reminder
+import com.hadiyarajesh.notex.ui.component.EmptyView
+import com.hadiyarajesh.notex.ui.component.LoadingProgressBar
+import com.hadiyarajesh.notex.ui.component.RetryItem
+import com.hadiyarajesh.notex.utility.Constants
+import com.hadiyarajesh.notex.utility.singleClickable
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun RemindersScreen(
     navController: NavController,
+    remindersViewModel: RemindersViewModel
 ) {
+    val scope = rememberCoroutineScope()
+    val context = LocalContext.current
+    val reminders = remember { remindersViewModel.reminders }.collectAsLazyPagingItems()
+
     Scaffold { innerPadding ->
         Column(
             modifier = Modifier
                 .fillMaxSize()
                 .padding(innerPadding),
-            horizontalAlignment = Alignment.CenterHorizontally,
-            verticalArrangement = Arrangement.Center
+            horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            Text(text = stringResource(id = R.string.welcome_message))
+            if (reminders.itemCount > 0) {
```

Comment:
Reminders are paging data. You don't need to handle item count here as i've already handled it in `private fun AllRemindersView()` as below

```kotlin
loadState.refresh is LoadState.NotLoading && loadState.append.endOfPaginationReached && reminders.itemCount < 1 -> {
                    item {
                        EmptyView(
                            modifier = Modifier
                                .fillParentMaxSize()
                                .padding(8.dp),
                            text = stringResource(id = R.string.empty_message),
                        )
                    }
                }
```